### PR TITLE
Update color variables on DSCO Breadcrumbs component

### PR DESCRIPTION
**Pull request by kelbyhawn**
Swaps out color variables from `shared/color.scss` with Primitive colors from `primitiveColors.css` on the Breadcrumbs component. 

👀 **DOTD note:** there might be Eyes diffs with this.

## Links
Jira ticket: [DESIGN2-199](https://codedotorg.atlassian.net/browse/DESIGN2-199)

## Testing story
Tested locally in Storybook

----

## Before - using `shared/colors.scss`
Before

## After - using `primitiveColors.css`
After

### CODE REVIEW

File: apps/src/componentLibrary/breadcrumbs/breadcrumbs.module.scss [hunk 1]
```diff
@@ -13,28 +13,28 @@
     justify-content: center;
     align-items: center;
     gap: 10px;
-    color: $light-black;
+    color: var(--neutral-base-black);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/breadcrumbs/breadcrumbs.module.scss:16]: black

File: apps/src/componentLibrary/breadcrumbs/breadcrumbs.module.scss [hunk 2]
```diff
@@ -13,28 +13,28 @@
...
     font-feature-settings: 'liga' off, 'clig' off;
     text-decoration: none;
     border-radius: 4px;
 
     // Current location styles
     &:last-child {
-      color: $light_primary_500;
+      color: var(--brand-teal-50);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/breadcrumbs/breadcrumbs.module.scss:23]: teal

File: apps/src/componentLibrary/breadcrumbs/breadcrumbs.module.scss [hunk 3]
```diff
@@ -13,28 +13,28 @@
...
       cursor: default;
     }
 
     // Hover, Pressed, Visited styles
     &:hover, :active, :visited {
       &:not(:last-child) {
-        color: $light_gray_800;
+        color: var(--neutral-gray-80);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/breadcrumbs/breadcrumbs.module.scss:30]: hover

**Review by levadadenys — APPROVED** [review]
LGTM! Thank you!

One thing I also wonder though is that should we maybe add a comment somewhere that if there we'll be some failing eyes tests - dotd let us know? (there probably should be none, though it's possible there might be some and it might be ok and not)

**Comment by kelbyhawn** [comment]
> LGTM! Thank you!
> 
> One thing I also wonder though is that should we maybe add a comment somewhere that if there we'll be some failing eyes tests - dotd let us know? (there probably should be none, though it's possible there might be some and it might be ok and not)

Good idea! I'll add one to the top of the PR.

**Pull request merged**